src/xdecode/comments.conf

```diff
@@ -38,3 +38,10 @@ xc_jne 0xAAAAAAAA						memtest pattern verify
 xc_mem_write 0x00000000					VISOR prep
 xc_mem_write 0x007FD588					TEA prep
 xc_pci_write 0x88000880, 0x00000002		MIST MCPX disablement
+
+xc_mem_write 0x0F680500					NV-PRAMDAC-NVPLL-COEFF
+xc_mem_write 0x0F68050C					NV-PRAMDAC-PLL-COEFF-SELECT
+xc_pci_write 0x8000036C					FSB
+xc_io_write 0x0000C004, 0x00000021		PIC
+xc_io_write 0x0000C008, 0x00000012		pic scratch reg
```

Comment:
These may be wrong, I'm using the terminology from Cromwell for these 2 PIC regs.

Reply:
Yeah lines 50 and 51 are a bit complicated as technically they depend on the I2C address set by the previous IO write to port 0xC004, so it's possible x-codes attempt to write to some other SMBus device with values 0x12 or 0x13 and it would get commented as PIC traffic incorrectly in this case. Since the commenting system doesn't support pattern matching across multiple lines, I'm inclined to have you remove these two for now. Thoughts?

Reply:
After reviewing my existing comments file I'm doing the same thing with the encoder however, so perhaps I deemed it fine if there's only a single match against all known retail/debug bios images. Merging, thanks!